packages/server/graphql/mutations/helpers/updateReflectionLocation/addReflectionToGroup.ts

```diff
@@ -63,7 +71,23 @@ const addReflectionToGroup = async (
     }).run()
 
     const nextTitle = getGroupSmartTitle(nextReflections)
-    await updateSmartGroupTitle(reflectionGroupId, nextTitle)
+    const oldGroupHasSingleReflectionCustomTitle =
+      oldReflectionGroup.title !== oldReflectionGroup.smartTitle && oldReflections.length === 0
```

Comment:
+1 why do you prefer to only update the new group's title if the old group contains 1 reflection? If the old group has two reflections and a custom title and the new group has a smart title, wouldn't we still want to use the custom title from the old group?

Reply:
@nickoferrall My logic is the following:
If a group has > 1 reflection and a custom title, e.g. `Oranges`. And we move one reflection out of that group, we consider that the reflection does not belong to that title (`Oranges`), and we don't need to apply same title to one more group, i.e. have two `Oranges` groups.